packages/ui/src/components/topbar.tsx

```diff
@@ -18,7 +18,7 @@ const Topbar = {
     return (
       <div
         className={cx(
```

Comment:
Looks like this should be using `cn` from the utilities, not `cx` directly from CVA.
```suggestion
        className={cn(
```

Reply:
Fixed